tests/sentry/issues/test_suspect_tags.py

```diff
@@ -0,0 +1,69 @@
+import datetime
+import time
+import uuid
+
+from sentry.issues.suspect_tags import (
+    get_suspect_tag_scores,
+    query_baseline_set,
+    query_selection_set,
+)
+from sentry.testutils.cases import SnubaTestCase, TestCase
+
+
+class SnubaTest(TestCase, SnubaTestCase):
```

Comment:
```suggestion
class SuspectTagsTest(TestCase, SnubaTestCase):
```
Same for test_suspect_flags.py